.gitignore

```diff
@@ -173,14 +173,14 @@ hs_err_pid*
 [Dd]ebug/
 [Bb]in/
 [Oo]bj/
-logs/
+dotnet_3/em/embedded/rest/SampleProjectDriver/logs/
 *.dll
 dotnet_3/*/Properties/
```

Comment:
noticed this just now. likely too generic, ie, what if a SUT has a module called `Properties`? should use full paths instead of `*`. `*` can be fine for specific files (eg, all ending with `.dll`), but usually not for folder, as who knows what SUT could have in their project structures